doc/LICENSES.md

```diff
@@ -39,6 +39,7 @@ Bellow follows a list of all code licenses used in Cardinal and linked submodule
 | MindMeld              | GPL-3.0-or-later      | |
 | Mog                   | CC0-1.0               | |
 | mscHack               | BSD-3-Clause          | |
+| Prism                 | BSD-3-Clause          | |
```

Comment:
artwork license should be added below (in the page) for new modules, otherwise it only gets more painful.
doesnt have to be in this PR, can be in a next one.

from what I saw the CC-SA version is not mentioned in the readme, so we will need to ask.